tests/crypto/crypto.cpp

```diff
@@ -45,13 +45,13 @@ void hash_to_scalar(const void *data, std::size_t length, crypto::ec_scalar &res
 }
 
 void hash_to_point(const crypto::hash &h, crypto::ec_point &res) {
-  crypto::ge_p2 point;
-  crypto::ge_fromfe_frombytes_vartime(&point, reinterpret_cast<const unsigned char *>(&h));
-  crypto::ge_tobytes(crypto::operator &(res), &point);
+  ge_p2 point;
+  ge_fromfe_frombytes_vartime(&point, reinterpret_cast<const unsigned char *>(&h));
+  ge_tobytes(crypto::operator &(res), &point);
```

Comment:
Why ?

Reply:
maybe a merge/rebase failure...have to check

Reply:
good question. A mod/unmod during my G++ fight

reverted

Reply:
same as above

Reply:
Same, something broke and needs fixing.

Reply:
OK I think I found and I'm right.

in src/crypto/crypto.cpp we have 


```C
namespace crypto {

  using std::abort;
  using std::int32_t;
  using std::int64_t;
  using std::size_t;
  using std::uint32_t;
  using std::uint64_t;

  extern "C" {
    #include "crypto-ops.h"
    #include "random.h"
  }
....

```

The file test/crypto/crypto.cpp directly includes src/crypto/crypto.cpp (which a bit "ugly" I think).
This inclusion seems to temporarily move crypto-ops definition to crypto namespace

In my PR "crypto-ops.h" is incuded earlier pushing the definitions out of any namespace, as it should
according to namespace-less definition to crypto-ops.c/crypto-ops.h . 

In conclusion I think this modifications should be keep.
(I also clean up my includes modification in the upcoming push update)

Reply:
Why does crypto.h now include crypto-ops.h ? Is it really needed ? You seem to add just new prototypes, so th enew include doesn't seem needed at first glance, and it's apparently causing this namepace change.

Reply:
new/moved API use ge_p3

Reply:
I've waded through this code, and I see now how it is. I'm fine with those changes.